python/xvapitch/xvapitch_model.py

```diff
@@ -401,19 +402,24 @@ def infer_using_vals (self, logger, plugin_manager, cleaned_text, sequence, lang
                 "emHappy": emHappy_pred.reshape((emHappy_pred.shape[0],emHappy_pred.shape[2])),
                 "emSad": emSad_pred.reshape((emSad_pred.shape[0],emSad_pred.shape[2])),
                 "emSurprise": emSurprise_pred.reshape((emSurprise_pred.shape[0],emSurprise_pred.shape[2])),
-                "text": [val.split("|") for val in cleaned_text],
+                "sequence": sequence,
```

Comment:
Seemed like a pre-xVAPitch format. This change might break an old plugin though.